README.md

```diff
@@ -0,0 +1,11 @@
+# How to preview the documentation changes locally?
+
+The documentation can hosted on a local webserver via httpd.
+
+You can run it with the following command, note that it requires docker:
+
+```shell
+sh start-preview.sh
```

Comment:
Can we make the script executable and call `./start-preview.sh`?

Reply:
Hi, does this means users have to run chmod +x themselves?

Reply:
Got it, I've updated the file permission and the readme